homeassistant/components/improv_ble/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain": "improv_ble",
+  "name": "Improv via BLE",
+  "bluetooth": [
+    {
+      "service_uuid": "00467768-6228-2272-4663-277478268000",
+      "service_data_uuid": "00004677-0000-1000-8000-00805f9b34fb"
```

Comment:
If we want to be backwards compatible with Improv via BLE servers which don't have this, we should not require it here but instead filter in the bluetooth step.

Reply:
I am highly in favor of not supporting it without the service data as it avoids future complaints about discovery flows that need to be ignored.  

It would be much more painful to change it later than to require it now